cli-tools/src/test/java/com/radixdlt/cloud/AWSSecretsTest.java

```diff
@@ -0,0 +1,89 @@
+ * it may be implemented or used or that it will be appropriate for any specific purpose.
+ *
+ * Neither the licensor nor any current or former employees, officers, directors, partners,
+ * trustees, representatives, agents, advisors, contractors, or volunteers of the Licensor
+ * shall be liable for any direct or indirect, special, incidental, consequential or other
+ * losses of any kind, in tort, contract or otherwise (including but not limited to loss
+ * of revenue, income or profits, or loss of use or data, or loss of reputation, or loss
+ * of any economic or other opportunity of whatsoever nature or howsoever arising), arising
+ * out of or in connection with (without limitation of any use, misuse, of any ledger system
+ * or use made or its functionality or any performance or operation of any code or protocol
+ * caused by bugs or programming or logic errors or otherwise);
+ *
+ * A. any offer, purchase, holding, use, sale, exchange or transmission of any
+ * cryptographic keys, tokens or assets created, exchanged, stored or arising from any
+ * interaction with the Work;
+ *
+ * B. any failure in a transmission or loss of any token or assets keys or other digital
+ * artefacts due to errors in transmission;
+ *
+ * C. bugs, hacks, logic errors or faults in the Work or any communication;
+ *
+ * D. system software or apparatus including but not limited to losses caused by errors
+ * in holding or transmitting tokens by any third-party;
+ *
+ * E. breaches or failure of security including hacker attacks, loss or disclosure of
+ * password, loss of private key, unauthorised use or misuse of such passwords or keys;
+ *
+ * F. any losses including loss of anticipated savings or other benefits resulting from
+ * use of the Work or any changes to the Work (however implemented).
+ *
+ * You are solely responsible for; testing, validating and evaluation of all operation
+ * logic, functionality, security and appropriateness of using the Work for any commercial
+ * or non-commercial purpose and for any reproduction or redistribution by You of the
+ * Work. You assume all risks associated with Your use of the Work and the exercise of
+ * permissions under this License.
+ */
+
+package com.radixdlt.cloud;
+
+import java.nio.file.Paths;
+import java.util.UUID;
+import org.junit.Test;
+
+public class AWSSecretsTest {
+  @Test
+  public void testExampleInvocation() {
+    var args =
+        new String[] {
+          "--enable-aws-secrets=false",
+          "--recreate-aws-secrets=false",
+          "--network-name=dumunet",
+          "--node-name-prefix=node",
+          "--node-number=0",
+          "--node-names=dumunet_ap_south_1_node0,dumunet_eu_west_1_node0,dumunet_eu_west_1_node1,dumunet_eu_west_1_node2,dumunet_eu_west_1_node3",
+          "--path="
+              + Paths.get(
+                  System.getProperty("java.io.tmpdir"),
+                  UUID.randomUUID().toString().replaceAll("[^a-zA-Z]", ""))
```

Comment:
do we already have the right deps here to do `@Rule public TemporaryFolder folder = new TemporaryFolder();` ?

If yes, then I recommend it.
(you will be able to do just `.newFile()` instead of inventing the "UUID name gen" + the file will auto-delete self)

Reply:
Yes, totally agree - thanks